modules/impactifyBidAdapter.md

```diff
@@ -3,7 +3,7 @@
 ```
 Module Name: Impactify Bidder Adapter
 Module Type: Bidder Adapter
-Maintainer: thomas.destefano@impactify.io
+Maintainer: filipe.neves@impactify.io
```

Comment:
Is there anyway to change this to a community email box instead of a single person?